sw/blas/gemm/verify.py

```diff
@@ -55,8 +59,8 @@ def main():
     c_golden = golden_model(1, a, b, beta, c).flatten()
 
     absolute_err = np.absolute(c_golden - c_actual)
-    fail = np.any(absolute_err > ERR_THRESHOLD)
-    if (fail):
+    fail = np.any(absolute_err > ERR_THRESHOLD[prec]) or np.isnan(c_actual).any()
```

Comment:
Good catch. I guess this might be needed also in the other kernels' verification scripts.
Should we define a reusable function in `data_utils.py`?

Reply:
That would be a good idea yes. I will add that